vuepress/docs/next/tutorials/consume/multitenancy-tutorial.md

```diff
@@ -0,0 +1,248 @@
+
+See <!-- [Multitenancy on Entando](../../docs/consume/multitenancy.md) -->, for details on concepts and architecture. 
+
+## Prerequisites
+* [A working instance of Entando 7.2 or higher.](../../../docs/getting-started/README.md)
+
+* Verify dependencies with the [Entando CLI](../../../docs/getting-started/entando-cli.md#check-the-environment): `ent check-env develop`
+
+## Configure the Services for the Primary Tenant
+
+1. Configure the Redis cache management service. Redis integration tutorial
+
+2. Add Solr integration for the external search engine. Solr integration tutorial
+
+3. Add CDS integration to manage static resources. CDS integration tutorial
+
+## Configure the Secondary Tenant
+The secondary tenant has the same capabilities as the primary tenant but with its own isolated data. For each new tenant, you will need to configure services for the Keycloak, database system, Solr core, and CDS instance. Each tenant will also require a ConfigMap.
+
+A subdomain name is required for each tenant, referred to as YOUR-TENANT1-CODE. This is the main way the tenants are identified internally. A secondary tenant can have multiple Fully Qualified Domain Names, (FQDNs) as long as they are defined in the `fqdns` field in the ConfigMap as shown in the example below. The `fqdns` field determines which tenant's configmap to use when an http request is made.
+
+### Keycloak Configuration
+Import the realm for Keycloak from the primary tenant and reconfigure it for the secondary tenant as follows. Note that YOUR-TENANT1-CODE is the subdomain name of your first secondary tenant.
+1. Go to your [Keycloak admin console](../../docs/consume/identity-management.md#authorization) which is typically located at `http://YOUR-HOST-URL/auth`.
+2. Follow this tutorial to [Create a Backup of the Keycloak Realm](../devops/backing-restoring-keycloak.md). 
+3. Edit the generated JSON file of the realm with these updates:
+     * remove every `id` attribute. There will be many. Save the JSON as `realm-YOUR-TENANT1-CODE.json, using the tenant code. The tenant code will become the prefix of the domain host name or URL. 
+     * Replace the properties `realm` & `displayName` with your tenant code.
+     * Add the tenant code as a prefix followed by "." to the following fields:
+`redirectUris` & `webOrigins` under the clientId for `entando-web`, `quickstart` and `quickstart-de`.  
+For example: 
+```
+"redirectUris": [
+    	"http://YOUR-TENANT1-CODE.YOUR-HOST-URL/entando-de-app/*"
+  	],
+```
+Save the file. 
+
+4. In the Keycloak admin console, go to `Import` in the left navigation sidebar to add the new realm for first tenant. Select the new JSON file for the first tenant. **Is this the primary's keycloak admin console??? And do you import realm or "Add realm" under Entando Header **
```

Comment:
@pietrangelo, when you import the primary tenant REALM for the secondary tenant's keycloak, do you "Add Realm" in the Drop Down arrow in the left navigation Bar?

Reply:
@jyunmitch you need to select `Add realm` in the Drop Down arrow in the left menu, then import the json definition of the new tenant realm.
The `Enabled` switch needs to be `ON` otherwise the new realm will be unavailable.